### YamlMime:PythonClass
uid: azure.ai.ml.exceptions.PipelineChildJobError
name: PipelineChildJobError
fullName: azure.ai.ml.exceptions.PipelineChildJobError
module: azure.ai.ml.exceptions
inheritances:
- azure.ai.ml.exceptions.MlException
summary: Exception that the pipeline child job is not supported.
constructor:
  syntax: 'PipelineChildJobError(job_id: str, command: str = ''parse'', prompt_studio_ui:
    bool = False)'
  parameters:
  - name: job_id
    isRequired: true
  - name: command
    defaultValue: parse
  - name: prompt_studio_ui
    defaultValue: 'False'
methods:
- uid: azure.ai.ml.exceptions.PipelineChildJobError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.ai.ml.exceptions.PipelineChildJobError.raise_with_traceback
  name: raise_with_traceback
  summary: 'Raise the exception with the existing traceback.


    *Deprecated since version 1.22.0:* This method is deprecated as we don''t support
    Python 2 anymore. Use raise/from instead.'
  signature: raise_with_traceback() -> None
- uid: azure.ai.ml.exceptions.PipelineChildJobError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.ai.ml.exceptions.PipelineChildJobError.args
  name: args
- uid: azure.ai.ml.exceptions.PipelineChildJobError.error_category
  name: error_category
  summary: Return the error category.
  return:
    description: The error category.
    types:
    - <xref:azure.ai.ml.exceptions.ErrorCategory>
- uid: azure.ai.ml.exceptions.PipelineChildJobError.no_personal_data_message
  name: no_personal_data_message
  summary: Return the error message with no personal data.
  return:
    description: No personal data error message.
    types:
    - <xref:str>
- uid: azure.ai.ml.exceptions.PipelineChildJobError.target
  name: target
  summary: Return the error target.
  return:
    description: The error target.
    types:
    - <xref:azure.ai.ml.exceptions.ErrorTarget>
- uid: azure.ai.ml.exceptions.PipelineChildJobError.ERROR_MESSAGE_TEMPLATE
  name: ERROR_MESSAGE_TEMPLATE
  signature: ERROR_MESSAGE_TEMPLATE = 'az ml job {command} is not supported on pipeline
    child job, {prompt_message}.'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.PROMPT_PARENT_MESSAGE
  name: PROMPT_PARENT_MESSAGE
  signature: PROMPT_PARENT_MESSAGE = 'please use this command on pipeline parent job'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.PROMPT_STUDIO_UI_MESSAGE
  name: PROMPT_STUDIO_UI_MESSAGE
  signature: PROMPT_STUDIO_UI_MESSAGE = 'please go to studio UI to do related actions{url}'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.continuation_token
  name: continuation_token
  signature: 'continuation_token: str | None'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.exc_msg
  name: exc_msg
  signature: 'exc_msg: str'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.exc_traceback
  name: exc_traceback
  signature: 'exc_traceback: TracebackType | None'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.exc_type
  name: exc_type
  signature: 'exc_type: Type[Any] | None'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.exc_value
  name: exc_value
  signature: 'exc_value: BaseException | None'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.inner_exception
  name: inner_exception
  signature: 'inner_exception: BaseException | None'
- uid: azure.ai.ml.exceptions.PipelineChildJobError.message
  name: message
  signature: 'message: str'
